commit 9a248c1ca916a956366b6c5e4fafa603d5d1ae9d
Author: Andreas Sturmlechner <andreas.sturmlechner@gmail.com>
Date:   Sun Jul 9 19:53:44 2017 +0200

    CMakeLists.txt: Minor improvements
    
    Summary:
    CMAKE_MINIMUM_REQUIRED first line
    Remove duplicate lines
    KF5DocTools optional
    subdirs -> add_subdirectory
    
    Reviewers: behrmann
    
    Differential Revision: https://phabricator.kde.org/D6588

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5b3eb2b..fd1ecf1 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,11 +1,10 @@
-#cmake file for building Kolor-Manager and related modules
+CMAKE_MINIMUM_REQUIRED(VERSION 3.0 FATAL_ERROR)
  
 PROJECT(kolor-manager)
 SET( PACKAGE_NAME kolor-manager )
 
 SET( CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_SOURCE_DIR}/cmake/modules )
 
-
 # project version
 SET( ${PROJECT_NAME}_MAJOR_VERSION 1)
 SET( ${PROJECT_NAME}_MINOR_VERSION 1)
@@ -13,10 +12,9 @@ SET( ${PROJECT_NAME}_MICRO_VERSION 1)
 
 SET( ${PROJECT_NAME}_VERSION ${${PROJECT_NAME}_MAJOR_VERSION}.${${PROJECT_NAME}_MINOR_VERSION}.${${PROJECT_NAME}_MICRO_VERSION} )
 
-FIND_PACKAGE(Qt5Widgets)
-CMAKE_MINIMUM_REQUIRED(VERSION 3.0)
 FIND_PACKAGE(ECM REQUIRED NO_MODULE)
 SET(CMAKE_MODULE_PATH ${ECM_MODULE_PATH} ${CMAKE_MODULE_PATH})
+
 INCLUDE(KDEInstallDirs)
 INCLUDE(KDECompilerSettings)
 INCLUDE(KDECMakeSettings)
@@ -24,7 +22,9 @@ INCLUDE(FeatureSummary)
 INCLUDE(GenerateExportHeader)
 
 FIND_PACKAGE(Qt5 REQUIRED COMPONENTS Core Widgets)
-FIND_PACKAGE(KF5 REQUIRED COMPONENTS ConfigWidgets CoreAddons I18n DocTools)
+FIND_PACKAGE(KF5 REQUIRED COMPONENTS ConfigWidgets CoreAddons I18n)
+
+FIND_PACKAGE(KF5DocTools)
 
 FIND_PACKAGE(Oyranos REQUIRED)
 FIND_PACKAGE(Synnefo REQUIRED)
@@ -39,11 +39,13 @@ LINK_DIRECTORIES(
 
 SET( ${CMAKE_BUILD_TYPE} Debug)
 
-SUBDIRS( devices
-         information
-         settings
-         doc
-       )
+ADD_SUBDIRECTORY( devices )
+ADD_SUBDIRECTORY( information )
+ADD_SUBDIRECTORY( settings )
+
+IF(KF5DocTools_FOUND)
+    ADD_SUBDIRECTORY( doc )
+ENDIF()
 
 
 INSTALL( FILES settings-kolor-management.desktop  DESTINATION  ${SERVICES_INSTALL_DIR} )
